src/commands/howto.ts

```diff
@@ -119,7 +125,9 @@ const postHowtoDm = async (command: HowtoCommand, targetUser: User) => {
     return;
   }
   let message = await targetUser.send(body);
-  await Promise.all(command.reactions.map(r => message.react(r.emoji)));
+  
+  // await 
+  Promise.all(command.reactions.map(r => message.react(r.emoji)));
```

Comment:
If this is crashing because of rate limiting (only crashes when reacting 4), try reacting sequentially instead of all at once.

```typescript
for (const r of command.reactions) {
  await message.react(r.emoji);
}
```

Reply:
I think I did,  but will recheck.

Reply:
I tried a couple of possibilities, and I think I observed that the main reason of the crash are not necessarily emojis, but the follow-up "please see your DMs" message. I made the two actions: "send DM with reactions" and "poke the user to see DMs" concurrent and now my development bot seems to consistently pass and does not crash.

Please verify if you think it's OK.